src/Sylius/Bundle/AdminBundle/Resources/config/routing/ajax/product.yml

```diff
@@ -41,5 +41,6 @@ sylius_admin_ajax_product_index:
         _controller: sylius.controller.product:indexAction
         _format: json
         _sylius:
+            serialization_groups: [Default]
```

Comment:
Could you add an upgrade note to `UPGRADE-1.8.md` mentioning this change and its effects?

Reply:
Added it 😉